src/styles/blocks/header.scss

```diff
@@ -0,0 +1,58 @@
+.header {
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    display: block;
+  }
+}
+
+.nav {
+  display: flex;
+  padding: 0;
+  margin: 0;
+
+  &__item {
+    list-style: none;
+  }
+
+  & > *:not(:first-child) {
+    margin-left: 20px;
```

Comment:
The left margin for `.nav > *:not(:first-child)` should be a multiple of 4. Consider changing the `margin-left` value to comply with the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).